blog-homepage/src/MissedArticleList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import missedArticles from './missed-articles.json';
+import Box from './Box';
```

Comment:
When possible, I'd recommend being more specific with variable names so that people looking at your code can quickly tell what things are. For example, `Article` as opposed to `Box`, or `months` as opposed to `m`. Just a nitpick though.